kahuna/app/controllers/FeatureSwitchController.scala

```diff
@@ -0,0 +1,52 @@
+package controllers
+
+import play.api.mvc.Cookie
+
+case class FeatureSwitch(key: String, title: String, default: Boolean)
+
+object ExampleSwitch extends FeatureSwitch(
+  key = "example-switch",
+  title = "An example switch. Use rounded corners for the feature switch toggle",
+  default = false
+)
+
+class FeatureSwitchController(featureSwitches: List[FeatureSwitch]){
```

Comment:
I'm possibly being a bit pedantic, but this class isn't a controller in the play terminology (a controller's methods would be `Action`s, listed in `conf/routes` to handle http requests, see <https://www.playframework.com/documentation/2.8.x/ScalaActions>). I think it'll be confusing to have this named as a controller and in the controller directory, so maybe we should call it a service (or even `FeatureSwitches`?) and move it to the lib directory instead

Reply:
That sounds very sensible - I'll do a restructure.

Reply:
(Now done)